piglow/piglow.go

```diff
@@ -0,0 +1,211 @@
+// Package piglow implements a driver for the Pimoroni PiGlow.
+package piglow
+
+import (
+	"golang.org/x/exp/io/i2c"
+	"golang.org/x/exp/io/i2c/driver"
+)
+
+// PiGlow represents a PiGlow device
+type PiGlow struct {
+	conn *i2c.Device
+}
+
+// Open opens a new PiGlow. A PiGlow must be closed if no longer in use.
+// If the PiGlow has not been powered down since last use, it will be opened
+// with it's last programmed state.
+func Open(o driver.Opener) (*PiGlow, error) {
+	conn, err := i2c.Open(o)
+	if err != nil {
+		return nil, err
+	}
+
+	return &PiGlow{
```

Comment:
return &PiGlow{conn:conn}, nil

Avoid whitespace if it is not helping the readability. These lines with additional whitespace are a bit unnecessarily verbose.